docs/dictionary/control_st/if.lcdoc

```diff
@@ -4,9 +4,30 @@ Type: control structure
 
 Syntax: if <condition> then <statementList> [else <elseStatementList>]
 
+Syntax:if <condition> then
+   <statementList>
+ [else if <condition> then
+   <elseifStatementList>]
+ [else 
+   <elseStatementList>]
+end if
+
+
 Summary:
-<execute|Executes> a list of <statement|statements> if a condition is
-true. 
+Evaluates a <conditional expression|condition> to determine if it is true
+and if so <execute|executes> a subsequent <statment|statement> or <statementlist|statement list>.
+
+If the <conditional expression|condition> is false and the <if> <control structure> contains
```

Comment:
this should be <condition|conditional expression> - so that it refers to the condition parameter but is rendered as 'conditional expression'.